### PR TITLE
Added Lesson and Subject classes

### PR DESCRIPTION
**Pull request by justinlengch**
Also deleted Tag class since I replaced it with Lesson

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Remark.java [hunk 1]
```diff
@@ -7,6 +7,8 @@
  * Guarantees: immutable; is always valid
  */
 public class Remark {
```
Comment (HusseinSafwan02) [inline src/main/java/seedu/address/model/person/Remark.java:9]: will we be including the remark class?

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 2]
```diff
@@ -23,21 +24,28 @@ public static Person[] getSamplePersons() {
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
                 new Address("Blk 30 Geylang Street 29, #06-40"),
-                    EMPTY_REMARK, getTagSet("friends")),
+                    new Subject("Math"),
+                    EMPTY_REMARK, getLessonSet("Maths|10-05-2004|12:29|0")),
```
Comment (HusseinSafwan02) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:28]: have to include test cases

Comment (justinlengch) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java:28]: Ok done!

**Review by leepoeaik — APPROVED** [review]
Looks good to me!

**Pull request merged**